File: Java/Program/Ram.java
package Java.Program;

public class Ram extends Komponen {
    // Attributes
    private int kapasitasGB;
    private String ddr;
    
    // Constructors
    public Ram() {
        this.kapasitasGB = 0;
        this.ddr = "";
    }
    
    public Ram(String merk, String nama, int kapasitasGB, String ddr) {
        super(merk, nama);
        this.kapasitasGB = kapasitasGB;
        this.ddr = ddr;
    }
    
    // Getters
    public int getKapasitasGB() {
        return this.kapasitasGB;
    }
    
    public String getDdr() {
        return this.ddr;
    }
    
    // Setters
    public void setKapasitasGB(int kapasitasGB) {
        this.kapasitasGB = kapasitasGB;
    }
    
    public void setDdr(String ddr) {
        this.ddr = ddr;
    }
}

File: Python/Program/Ram.py
from Komponen import Komponen

class Ram(Komponen):
    # Constructor
    def __init__(self, merk="", nama="", kapasitas_GB=0, ddr=""):
        super().__init__(merk, nama)
        self.kapasitas_GB = kapasitas_GB
        self.ddr = ddr
    
    # Getters
    def get_kapasitas_GB(self):
        return self.kapasitas_GB
    
    def get_ddr(self):
        return self.ddr
    
    # Setters
    def set_kapasitas_GB(self, kapasitas_GB):
        self.kapasitas_GB = kapasitas_GB
    
    def set_ddr(self, ddr):
        self.ddr = ddr
